Dockerfile

```diff
@@ -1,32 +1,65 @@
-FROM node:0.12
+# Build:
+# docker build -t lirantal/meanjs .
+#
+# Run:
+# docker run -it lirantal/meanjs
+
+FROM ubuntu:latest
+MAINTAINER Liran Tal <liran.tal@gmail.com>
+
+# Install Utilities
+RUN apt-get update -q
+RUN apt-get install -yqq wget aptitude htop vim vim-puppet git traceroute dnsutils curl ssh sudo psmisc gcc make build-essential libfreetype6 libfontconfig tree tcpdump nano
```

Comment:
In that light this list might be fine!

BTW Check note about "Sort multi-line arguments" from https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/